lib/extensions.js

```diff
@@ -0,0 +1,75 @@
+// Imports
+const fs = require('fs')
+const path = require('path')
+const foundationExtensions = require('../app/config').foundationExtensions || ['govuk-frontend']
+
+// Utils
+const pathFromRoot = (...all) => path.join.apply(null, [__dirname, '..'].concat(all))
+const pathToHookFile = packageName => pathFromRoot('node_modules', packageName, 'govuk-prototype-kit.config.json')
+const objectMap = (object, mapFn) => Object.keys(object).reduce((result, key) => {
+  result[key] = mapFn(object[key], key)
+  return result
+}, {})
+const removeDuplicates = arr => arr.filter((i, pos, self) => self.indexOf(i) === pos)
```

Comment:
Personally I find both approaches quite hard to read, but there's another approach, using `set` here:

https://stackoverflow.com/a/9229821/54221

Reply:
alternatively, if we never want duplicates, we could use a `set` from the start

Reply:
Do we use a library which has this feature?  Could we add it to some helpers so that this file just uses a "removeDuplicates" function rather than needing to define it?